basketball_reference_web_scraper/client.py

```diff
@@ -212,6 +212,35 @@ def team_box_scores(day, month, year, output_type=None, output_file_path=None, o
     )
     return output_service.output(data=values, options=options)
 
+def get_roster(team, year=None, output_type=None, output_file_path=None, output_write_option=None, json_options=None):
+    try:
+        http_service = HTTPService(parser=ParserService())
+        if year == None:
+            today = datetime.now()
+            year = today.year
+            if today.month >=7:
+                year += 1
+        if len(team) > 3:
+            team=TEAM_TO_TEAM_ABBREVIATION[team.upper()]
+        values=http_service.get_team_roster(team=team, year=year)
+    except requests.exceptions.HTTPError as http_error:
+        if http_error.response.status_code == requests.codes.not_found:
+            raise InvalidTeam(team=team, year=year)
```

Comment:
I think this error's name is slightly inaccurate - the `team` value could be valid, but the `season end year` value could be invalid, like https://www.basketball-reference.com/teams/BOS/1020.html

I'd prefer to call this error `InvalidTeamSeason` (or something similar).

(Note that I've made similar inaccurate naming mistakes in other methods, like `InvalidDate`, that need to be corrected in the future.)